wp-content/mu-plugins/register-rest-routes.php

```diff
@@ -135,50 +136,50 @@
   register_rest_route($v, '/locations/', array(
     'methods' => 'GET',
     'callback' => function(WP_REST_Request $request) use ($transients, $exp) {
-      $transient = $transients['location'];
-      $data = get_transient($transient);
+      global $sitepress;
+      $default_lang = $sitepress->get_default_language();
+      $data = array( );
```

Comment:
@theasteve the transient has been removed here. Everything else looks great, but should be added back in order to approve this PR.

This is important for caching the request. It will retrieve the request from the cache as opposed to running the query every time this enpoint is hit. This speeds up the request quite significantly.

You probably were wondering why you weren't seeing data while working on this. The cache needs to be flushed to see new data. You could also try impelmenting a cache busting custom query param to do your debugging.

Hit me up with any Qs on this.